packages/api/helper/googleSheetsAPIConnection.js

```diff
@@ -0,0 +1,33 @@
+const { google } = require('googleapis');
+const spreadsheetsIdPar = '1GGtT9THPQZBHJSFcQHNu1mYBOdRLeBHZjTz0B5gcY2U';
```

Comment:
is this the Google Sheet ID for the Nominations? This should be an Env variable as well then